localgov_page_components.install

```diff
@@ -24,3 +24,31 @@ function localgov_page_components_update_8001(&$sandbox) {
 
   return t('Please export your sites configuration! Config entities for localgov_page_components where updated.');
 }
+
+function localgov_page_components_install($is_syncing) {
```

Comment:
Hi Matt,
It would be good to have a check here for the presence of the LinkIt module.  Something like the following snippet should do:

```
if (!Drupal::service('module_handler')->moduleExists('linkit')) {
  return;
}
```